portfolio/pom.xml

```diff
@@ -29,6 +29,12 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+
```

Comment:
run mvn tidy:pom to tidy up the formatting

Reply:
I have just run it. Thank you for giving the tips!